adapters/AppLovin/AppLovinAdapter/GADMediationAdapterAppLovin.m

```diff
@@ -121,14 +121,6 @@ - (void)collectSignalsForRequestParameters:(nonnull GADRTBRequestParameters *)pa
                          completionHandler:
                              (nonnull GADRTBSignalCompletionHandler)completionHandler {
   [GADMAdapterAppLovinUtils log:@"AppLovin adapter collecting signals."];
-  // Check if supported ad format.
```

Comment:
@tiandrew , we should keep this check in right? We are essentially making sure we don't return a signal for Native.